lib/sweet_xml.ex

```diff
@@ -359,6 +371,40 @@ defmodule SweetXml do
     end)
   end
 
+  def stream_tags!(doc, tags, options \\ []) do
```

Comment:
I'll check back on this and `stream!` later, but a first glance I don't see any problem.